test/specs/commonTests.js

```diff
@@ -858,31 +858,13 @@ export const implementsImageProp = (Component, options = {}) => {
  * @param {Object} [options.requiredProps={}] Props required to render the component.
  */
 export const implementsTextAlignProp = (Component, options = {}) => {
-  const { requiredProps = {} } = options
   const { assertRequired } = commonTestHelpers('implementsTextAlignProp', Component)
 
   describe('aligned (common)', () => {
     assertRequired(Component, 'a `Component`')
 
     _noDefaultClassNameFromProp(Component, 'textAlign', options)
     _noClassNameFromBoolProps(Component, 'textAlign', options)
-
-    _.each(Component._meta.props.aligned, (propVal) => {
```

Comment:
That would also work.  I checked all the usages and all components that implement `TEXT_ALIGNMENTS` implement all of them.  I'm OK with the default pattern as well.